bin/run-suite

```diff
@@ -6,6 +6,65 @@ import subprocess
 import sys
 import copy
 import timeit
+import io
+import argparse
+
+app_list = {
+      "2DConvolution",
+      "2mm",
+      "3DConvolution",
+      "3mm",
+      "arith",
+      "atax",
+      "bicg",
+      "blocked_transform",
+      "correlation",
+      "covariance",
+      "dag_task_throughput_independent",
+      "dag_task_throughput_sequential",
+      "DRAM",
+      "fdtd2d",
+      "gemm",
+      "gesummv",
+      "gramschmidt",
+      "host_device_bandwidth",
+      "kmeans",
+      "lin_reg_coeff",
+      "lin_reg_error",
+      "local_mem",
+      "matmulchain",
+      "median",
+      "mol_dyn",
+      "mvt",
+      "nbody",
+      "pattern_L2",
+      "reduction",
+      "scalar_prod",
+      "segmentedreduction",
+      "sf",
+      "sobel",
+      "sobel5",
+      "sobel7",
+      "syr2k",
+      "syrk",
+      "vec_add"
+    }
+
+known_fail = {"3DConvolution",
+      "DRAM",
+      "blocked_transform"}
+
+is_bandwidth_bench = {
+      "arith",
+      "blocked_transform",
+      "dag_task_throughput_independent",
+      "dag_task_throughput_sequential",
+      "DRAM",
+      "host_device_bandwidth",
+      "local_mem",
+      #"matmulchain",
```

Comment:
Why is it commented out? is it a bandwidth bench or not?

Reply:
Could have been of interest, fixed